lua_tests/gambiarra/gambiarra.lua

```diff
@@ -146,13 +147,22 @@ return function(name, f, async)
       f(handler, name, ...)
     end
 
+    local function cbError(err)
+      err = err:match(".-([^\\/]*)$") -- cut off path of filename
+      if not err:match('_*_TestAbort_*_') then
```

Comment:
What's this?

Reply:
That is the error callback which is set in `node.setonerror(cbError)` further down which is called as new catchall errorhandler instead of rebooting the chip. Its a yet undocumented feature by Terry. Read here for more info: https://gist.github.com/TerryE/646407100785fb00c2cbddbb0e8a2b61#panic-handling

The check for '_*_TestAbort_*_' is used to distinguish failures from `ok`, `nok` or `fail` from other application filures or lua `error()` calls.